static/includes/apps/SMBLimitations.md

```diff
@@ -0,0 +1,149 @@
+&NewLine;
+
+{{< expand "SMB Limitations" "v" >}}
+
+#### Windows, Linux, and MacOS Client SMB Limitations 
+{{< truetable >}}
+|  | Windows | Linux (Ext4) | Linux (XFS) | MacOS |
+|--|---------|--------------|-------------|-------|
+| Maximum file name length (characters) | 255 | 255 | 255 | 255 |
+| Maximum share name length (characters) | 255 | 255 | 255 | 255 |
+| Maximum length for path (characters) | 260* | 4096 | 4096 | 255*** |
+| Invalid characters | angle brackets < ><br>colon :<br>asterick *<br>slashes / \<br>pipe | | NULL (?0)<br>slash (/) | NULL (?0)<br>slash (/) | slash /<br>colon :<br> |
+| Case sensitivity | Yes | No | No | Yes |
+| Single file size/write size** | 16 TB | 16 TB | 8 EB | # |
+| Maximum volume size | 256 TB | 1 EB | 8 EB | # |
+| Available file descriptors | 1024 | 1024 | 1024 | 256 |
+| File permissions: ACL entries (ACEs) per ACL, per file or directory | 1025 | 32 | 1000+ | 64 |
```

Comment:
This isn't isn't related to SMB protocol.

Reply:
I'll strip out this information.